### DISCUSSION TITLE
Mutual fund market price changed to $0 and won't update

### DISCUSSION THREAD

**Discussion by FreeMyNook** [afadil/wealthfolio#189] [Q&A]
I've been evaluating Wealthfolio for a few months to better visualize my US-based retirement and investment accounts (think 401a/k, 403b, Roth IRA, brokerage).  I have successfully imported nearly 900 investment transactions dating back to December 2020.  I imported most of the transactions as TRANSFER_IN via CSV files, since I only hold mutual funds and ETFs (virtually, no cash).

In mid-November, I sold some shares of my largest holding (FXAIX) as a part of a portfolio rebalance.  Since entering that transaction, Wealthfolio has only reported the market price of FXAIX as $0.00, which throws off everything else in the tool.  Forcibly updating the portfolio from the main screen does nothing.  I've even updated the app (up to the current version for Windows) without any change in behavior.

I've tried deleting and re-entering transactions (as DEPOSIT, BUY, and SELL) and and even started over from scratch (by deleting/renaming app.db), but nothing I've tried has resulted in Wealthfolio working for me as expected.

Any thoughts?

![wealthfolio01](https://github.com/user-attachments/assets/717eb0c8-2540-449e-bc95-ca1e2f1200a6)
![wealthfolio02](https://github.com/user-attachments/assets/268713bc-fe68-484c-a820-a4601894a51a)

**Comment by FreeMyNook** [comment]
Not an answer, but I think I found a clue in the logs:
...
[2024-12-15][22:40:35][wealthfolio_core::portfolio::holdings_service][ERROR] Error fetching quote for symbol FSMDX: Record not found
[2024-12-15][22:40:35][wealthfolio_core::portfolio::holdings_service][ERROR] Error fetching quote for symbol FSSNX: Record not found
..
[2024-12-19][18:07:22][wealthfolio_core::portfolio::holdings_service][ERROR] Error fetching quote for symbol FXAIX: Record not found
...
[2024-12-20][15:58:56][wealthfolio_core::portfolio::holdings_service][ERROR] Error fetching quote for symbol FUAMX: Record not found

May be worth submitting an issue now that I see this.

**Reply by wliu88ca** [reply]
i got same issue
